src/inmanta/protocol/rest/model.py

```diff
@@ -0,0 +1,26 @@
+"""
+    Copyright 2020 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+from inmanta.data.model import BaseModel
+
+
+class HtmlContentType(BaseModel):
```

Comment:
I see. To cleanly fit this in the typing is kind of messy. 

`ReturnType` is completely generic, so it would support this. 
The handle annotation however is expects `Apireturn`, which is  `Union[int, ReturnTupple, "ReturnValue[ReturnTypes]", "ReturnValue[None]", ReturnTypes]` 
ReturnTypes is refined into types that are used all over the place, so should not be updated.

So, what we do here is not in line with the intent of the current typing (which is intend to offer a typed api backed by json). We either have to reconsider the api or shoehorn it in. In the shoehorning category, this is the cleanest solution.